legislators-current.yaml

```diff
@@ -817,7 +817,7 @@
     state: NJ
     party: Democrat
     class: 1
-    url: http://menendez.senate.gov
+    url: http://www.menendez.senate.gov
```

Comment:
This edit is good. The other 3 changes above in this file should be reverted. I can submit a PR to the PR.